### ISSUE TITLE
Make target install failing on Linux

### ISSUE THREAD

**Issue by pmattern** [team-charls/charls#12]
After successfully compiling latest VCS checkouts of CharLS on Linux make target `install` fails, apparently as it's expecting one of the source files at a location where it doesn't exist
```
# make install 
[ 42%] Built target CharLS
[100%] Built target charlstest
Install the project...
-- Install configuration: ""
-- Installing: /usr/lib/libCharLS.so.1.0
[...]
-- Installing: /usr/include/CharLS/losslesstraits.h
-- Installing: /usr/include/CharLS/scan.h
CMake Error at cmake_install.cmake:68 (file):
  file INSTALL cannot find
  "/usr/local/peter/Tests/CharLS/Projekt/charls/src/config.h".


Makefile:83: recipe for target 'install' failed
make: *** [install] Error 1
```

**Comment by pmattern** [comment]
Thanks for the quick fix.

While at it I'd like to ask whether there's any chance you could set another annotated tag?
The problem is that e. g. `git describe` is still yielding output like
```
$ git describe --always 
PreCpp11-183-g7ef088d
```
which is pretty misleading as the code does support C++ 11 since long time by now.

(The fact that only header files `charls.h` and `publictypes.h` are left is expected behaviour, isn't it?)

**Comment by vbaderks** [comment]
PreCpp11 is just the last tag that was set. CharLS on the master brach actually uses C++ 14. I created a new release tag, with just the version number to prevent the "misleading".

Yes, only charls.h and publictypes.h are the only header files needed to use the library. The other header files are internal and don't need to be "installed".

**Comment by pmattern** [comment]
Besides the tagged commit is in master the output of `git describe` in master remains the of the same kind
```
$ git describe 
PreCpp11-186-g73463f4
```
Was it by any chance a so-called [lightweight](https://git-scm.com/book/en/v2/Git-Basics-Tagging#Creating-Tags) tag only and not an annotated one?

**Comment by vbaderks** [comment]
I used github "create a release". This seems to only create a lightweight tag. Re-created the tag using the command line. git describe should now work.

**Comment by pmattern** [comment]
Confirmed. The tag may be considered a bit lengthy but that aside everything is alright now, IMHO
```
$ git branch --all -v
* master                     da6bb90 Fixed 1 clang -fsanitize warning and supressed 1 (code optimalization). Closes #11.
  remotes/projekt/1.x-master 82c0339 Merge branch '1.x-master' of https://github.com/team-charls/charls.git
  remotes/projekt/HEAD       -> projekt/master
  remotes/projekt/master     da6bb90 Fixed 1 clang -fsanitize warning and supressed 1 (code optimalization). Closes #11.

$ git describe 
2.0.0-alpha001-3-gda6bb90
```
